lib/elixir/lib/file.ex

```diff
@@ -1340,6 +1355,28 @@ defmodule File do
   defp normalize_modes([], true), do: [:binary]
   defp normalize_modes([], false), do: []
 
+  defp normalize_write_modes(modes) do
+    normalize_modes([:write] ++ modes, true)
```

Comment:
Here I'm setting `:binary` because, even though I'm not sure it's necessary, it's more like what the code used to do. We previously used `:file.write_file/3`, and [the docs say](http://erlang.org/doc/man/file.html#write_file-3) that "the mode flags binary and write are implicit" when calling that function.